kickoff/firefoxdetails.py

```diff
@@ -0,0 +1,784 @@
+from firefoxversions import firefoxversions as versions
+
+primary_builds = {
```

Comment:
We need a few things:
* the shipped date
* the current aurora version (we don't have this information in ship-it yet)
* a way to differentiate ESR & ESR_NEXT (currently 31 & 38)
* list of supported languages
* for all the locales, which versions (aurora, beta, release) are supported

About the data which are used by endpoints, it is mainly version name + shipped date.